packages/backfill/src/audit.ts

```diff
@@ -58,7 +57,7 @@ export function initializeWatcher(
     internalCacheFolder
   ]);
   watcher = chokidar
-    .watch(hashGlobs, {
+    .watch(["**"], {
```

Comment:
Is this correct?

Reply:
@bweggersen I don't know what to think of this.

Reply:
I don't understand why we are we audit only the files described by the hashGlobs. Can you help me here?

Reply:
This used to be `repositoryRoot`. And then I moved `repositoryRoot` to the options object as the `cwd`. At that time `hashGlobs` was `["**/*", "!**/node_modules/**", `!${outputFolder}/**`]`, so it included everything except for some things that could be ignored. But that was a mistake on my part, because I coupled audit to something that was a bit unrelated, especially as the code has evolved since then. `**` seems right. We might want to add `node_moduels` to the ignoreGlobs, or not. Maybe we want to make note of those changes as well.

Reply:
I don't know, what should we do? Any reason we should treat node_modules differently than other gitignored folders? The correct thing is probably to have a gitignore option in chokidar but that does not exist. :(

Reply:
The whole chokidar is suspect. Why not use git itself to figure out what is modified? Since you've already moved to using that as your change semantics , why not do that instead?

Reply:
I feel like that getting rid of hashGlob / outputGlob have negated the need to do audits at all. Is that right?

Reply:
* `**` is a probably right
* I don't think having a gitignore option in chokidar would be the right thing here, because you want to see every file that your package touches.
* We should still support audits. The purpose of the audit is to look at every file change caused by the package and report any file change outside of the package.
* I'm unsure if the check below is correct (unchanged by this PR). In Midgard our outputGlob is `["**/*", "!node_modules", "!package.json"]`. Maybe what's missing from the check below is passing a `cwd` to anymatch with the package path? E.g. if your package changed something outside of that package, it would still match `**/**/*` and so this file change would be marked as `changedFilesInsideScope`. But with cwd, I don't think they would be a match.  So, chokidar looks at the repo level and anymatch filters at the package level?

```

!anymatch(
    outputGlob.map(glob => path.posix.join("**", glob)),
    filePath
)
```